db/migrate/20230512094803_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorid
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    add_index :posts, :authorid
+  end
+end
```

Comment:
- [ ] Here also we have a duplicated file, let us kindly resolve it. It causes problems in migration